core/src/main/java/org/jruby/api/Convert.java

```diff
@@ -593,6 +596,18 @@ public static RubyInteger toInteger(ThreadContext context, IRubyObject arg) {
         return arg.convertToInteger();
     }
 
+    /**
+     * Create a Java String from a ByteList with the specified encoding.
+     * @param bytes to be made into a string
+     * @param encoding use encodings charset to make the String
+     * @return a new Java String
+     */
+    public static String asJavaString(ByteList bytes, Encoding encoding) {
```

Comment:
ByteList has Encoding, why is a second encoding passed in here?

Reply:
As it turns out none of the callers have a different encoding so I just make this a single param method.